utils/serialise.js

```diff
@@ -1,12 +1,13 @@
 module.exports = serialise
 
-function serialise (type, attributes, id) {
+function serialise (type, attributes, id, createdAt) {
   if (!type || !attributes) {
     throw new Error('Serialisation must include a type and some attributes.')
   }
   var data = {
     type: type,
-    attributes: attributes
+    attributes: attributes,
+    createdAt: createdAt
   }
```

Comment:
we don’t want to set other attributes than `type`, `id` and `attributes`, this is part of the [JSON API specification](http://jsonapi.org/). `createdAt` is an attribute like username, so I would set it directly on `attributes.createdAt`. 

What I think we could to do is to set `options.createdAt` in `lib/sign-up.js` before we pass it to `internals.serialise`.